Atomic/syscontainers.py

```diff
@@ -1954,6 +1954,15 @@ def _check_system_oci_image(self, repo, img, upgrade, src_creds=None):
             if not missing_layers:
                 return False
 
+        try:
+            d = util.Decompose(img)
+            if d.registry == "":
+                util.write_err("The image `{}` is not fully qualified.  The default registry configured for Skopeo will be used.".format(img))
+        except:  # pylint: disable=bare-except
```

Comment:
Hey, not a review, but a question here as I am relatively new to the code base. Hopefully you won't mind :p. I noticed that some of the expect code part, e.g: https://github.com/giuseppe/atomic/blob/4c45bee25c3bf23cce0d61143daf59bf2b58333f/Atomic/syscontainers.py#L865
does not have '#pylint: disable=bare-except' as a comment, so just wondering what will be the reason for this comment to be there? :).

Reply:
we don't specify an exception type here, so pylint would complain:

http://pylint.blogspot.com/2014/08/no-exception-types-specified-pylint.html

But here I really want to be sure no exceptions are raised ever, it is only a warning, in the worse case we will just miss it.

Reply:
ah I see, makes sense :-)